clang/test/dpct/device_global/device_global.cu

```diff
@@ -0,0 +1,104 @@
+// RUN: dpct --use-experimental-features=device_global -in-root %S -out-root %T/device_global %S/device_global.cu --cuda-include-path="%cuda-path/include" -- -std=c++14 -x cuda --cuda-host-only
```

Comment:
please check if --use-experimental-features=all works for this lit test.

Reply:
works